.expeditor/scripts/build_mac_component.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/local/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared.sh
+
+# Get secrets!
+eval "$(vault-util fetch-secret-env)"
+
+export HAB_AUTH_TOKEN="${ACCEPTANCE_HAB_AUTH_TOKEN}"
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
+
+########################################################################
+
+# `component` should be the subdirectory name in `components` where a
+# particular component code resides.
+#
+# e.g. `hab` for `core/hab`, `plan-build` for `core/hab-plan-build`,
+# etc.
+component=${1:?You must specify a component value}
+
+channel=$(get_release_channel)
+
+echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
+
+echo "--- Installing buildkite agent"
+brew tap buildkite/buildkite
+brew install --token="$BUILDKITE_AGENT_ACCESS_TOKEN" buildkite-agent
+
+echo "--- Installing mac bootstrap package"
+bootstrap_package_version="1.0.7"
+curl "https://packages.chef.io/files/current/mac-bootstrapper/${bootstrap_package_version}/mac_os_x/10.12/mac-bootstrapper-${bootstrap_package_version}-1.dmg" -o "mac-bootstrapper-${bootstrap_package_version}-1.dmg"
+sudo hdiutil attach "mac-bootstrapper-${bootstrap_package_version}-1.dmg"
+sudo installer -verbose -pkg "/Volumes/Habitat macOS Bootstrapper/mac-bootstrapper-${bootstrap_package_version}-1.pkg" -target /
+sudo hdiutil detach "/Volumes/Habitat macOS Bootstrapper"
+brew install wget
+export PATH=/opt/mac-bootstrapper/embedded/bin:/usr/local/bin:$PATH
+
+install_latest_hab_binary "$BUILD_PKG_TARGET"
+import_keys
+
+# Because we are not running as root on mac, move our keys
```

Comment:
Perhaps a little additional clarity 
```
# We invoke hab-plan-build.sh directly via sudo, so we don't get the key management that studio provides. 
# Copy keys from the user account Habitat cache to the system Habitat cache so that they are present for root.
```

Reply:
👍